corebehrt/modules/setup/directory_causal.py

```
@@ -116,7 +119,10 @@ def setup_train_xgb(self) -> None:
         # Write config in output directory.
         self.write_config(out, source="encoded_data", name=ENCODE_CFG)
         self.write_config(out, source="calibrated_predictions", name=CALIBRATE_CFG)
-        self.write_config(out, source="cohort", name=COHORT_CFG)
+        try:
+            self.write_config(out, source="cohort", name=COHORT_CFG)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)